### PR TITLE
Add handlers and renderers for docs and methods

### PR DESCRIPTION
**Pull request by Varanas**
This is the Julia side for supporting docs-at-a-keystroke in Atom.

### CODE REVIEW

File: src/eval.jl [hunk 1]
```diff
@@ -99,3 +99,21 @@ handle("eval-repl") do data
     end
   end
 end
+
+handle("docs") do data
+  @dynamic let Media.input = Editor()
+    result = withpath(getpath(data)) do
```
Comment (spencerlyon2) [inline src/eval.jl]: Why do we need the `withpath(getpath(` here?

Comment (Varanas) [inline src/eval.jl]: Because I have no idea how the docsystem works and just copied that from above.
It's probably unnecessary, but I'm not sure.

**Comment by one-more-minute** [comment]
Ok, so you're just treating doc requests as if the user had evaluated `@doc X`? Sounds like a reasonable approach as a first pass.

File: src/eval.jl (hunk 1 above)
Comment (one-more-minute) [inline src/eval.jl]: No, I don't think it should be necessary in this case, nor the Media.input binding. Also, on Julia 0.4 you can switch `@d` for just `d`.

**Comment by Varanas** [comment]
Yeah. I don't have any specific ideas for another approach, actually, besides the possibility to open a new Blink or browser window to display the online docs (like matlab does).

**Pull request merged**